FileSystem/Setup.js

```diff
@@ -0,0 +1,72 @@
+/**
+ * Creates a menu entry in the Google Docs UI when the document is opened.
+ * This method is only used by the regular add-on, and is never called by
+ * the mobile add-on version.
+ *
+ * @param {object} e The event parameter for a simple onOpen trigger. To
+ *     determine which authorization mode (ScriptApp.AuthMode) the trigger is
+ *     running in, inspect e.authMode.
+ */
+function onOpen(e) {
+  SpreadsheetApp.getUi().createAddonMenu()
```

Comment:
createMenu

Reply:
Actually this is a trigger, I don't think it can be renamed?

Reply:
Removed this completely